### PR TITLE
Fixed click on ADEME link

### LINKED ISSUE #2968: You can't click the link to the ADEME website
image
You can't click the link to the ADEME website in the Environment tab.

### PR DESCRIPTION
**Pull request by mrudultora**
####  Description
Added LinkMovementMethod on textview to open the link of ADEME website.

#### Related issues
Fixes #2968 

#### Related PRs
None

#### Screenshots

### CODE REVIEW

**Comment by teolemon** [comment]
Is it possible to make it generic for all links, regardless of their names ?

**Comment by mrudultora** [comment]
> Is it possible to make it generic for all links, regardless of their names ?

@teolemon If we talk about that particular textview, this will work for all the links in that textview. For other textviews, we have to add that feature.
The if block ( `if (environmentInfocard.contains("ADEME"))`  ) can be removed, if we want it to work for all the links. (whether it contains ADEME or not).

**Comment by mrudultora** [comment]
@teolemon Also, on clicking the link default browser of phone will open. I tried to open that in webview but was unable to do so. Could you help me or suggest me a way to open that in webview ? Or, is it fine to open this link in default browser?

**Comment by teolemon** [comment]
This field is remotely updated with methodology, so yes, we'll have various websites in the future, so the if block should be removed. 
There's no spam risk since it's not user editable

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/features/product/view/environment/EnvironmentProductFragment.kt [hunk 1]
```diff
@@ -94,6 +95,8 @@ class EnvironmentProductFragment : BaseFragment() {
         val environmentInfocard = product.environmentInfoCard
         if (!environmentInfocard.isNullOrEmpty()) {
             binding.environmentInfoText.append(HtmlCompat.fromHtml(environmentInfocard, HtmlCompat.FROM_HTML_MODE_COMPACT))
+            if (environmentInfocard.contains("ADEME"))
+                binding.environmentInfoText.movementMethod = LinkMovementMethod.getInstance()
```
Comment (teolemon) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/features/product/view/environment/EnvironmentProductFragment.kt]:
```suggestion
            binding.environmentInfoText.movementMethod = LinkMovementMethod.getInstance()
```

**Comment by mrudultora** [comment]
Changes done ! I had removed the if condition. Now it will work for every link.

**Review by teolemon — APPROVED** [review]

**Review by VaiTon — APPROVED** [review]

**Pull request merged**